docs/organizations/managing-security-and-risk.md

```diff
@@ -1,73 +1,148 @@
+To access the findings page with the corresponding filter applied, click on a number.
 
-![Security and risk management items](images/security-risk-management-item-list.png)
+![Security and risk management open findings overview panel](images/security-risk-management-overview-open.png)<!-- TODO TAROT-2591: Screenshot -->
 
-## Exporting the security item list
+### Open finding distribution
+
+The **Open finding distribution** panel displays the relative distribution of open findings by scan kind, severity, or status.
+
+To select the desired distribution, use the drop-down in the top right-hand corner of the panel.
+
+To access the findings page with the corresponding filter applied, click on a number.
+
+![Security and risk management open finding distribution panel](images/security-risk-management-overview-distribution.png)<!-- TODO TAROT-2591: Screenshot -->
+
+### Open finding history
+
+The **Open finding history** panel displays a graph showing weekly trends of open findings from the past three months, grouped by severity.
+
+For a detailed view of the distribution on a specific week, hover over the graph.
+
+![Security and risk management open finding history panel](images/security-risk-management-overview-history-open.png)<!-- TODO TAROT-2591: Screenshot -->
+
+### Activity history
+
+The **Activity history** panel displays a graph showing weekly counts of open and closed findings over the past three months and the overall open finding trend.
+
+For a detailed view of the counts on a specific week, hover over the graph.
+
+![Security and risk management activity history panel](images/security-risk-management-overview-history-activity.png)<!-- TODO TAROT-2591: Screenshot -->
+
+### Top 10 high-risk repositories
+
+The **Top 10 high-risk repositories** panel displays the repositories with the highest number of open findings, ordered by severity.
+
+!!! note
+    This panel may list less than ten repositories if there are fewer than ten repositories with open findings in the organization or if less than ten repositories are selected in the dropdown filter.
+
+![Security and risk management high risk repositories panel](images/security-risk-management-overview-top-risk.png)<!-- TODO TAROT-2591: Screenshot -->
+
+### Top 10 common security categories
+
+The **Top 10 common security categories** panel displays the most common security categories of open findings, ordered by count.
+
+To access the findings page with the corresponding filter applied, click on a category.
+
+![Security and risk management top categories panel](images/security-risk-management-overview-top-categories.png)<!-- TODO TAROT-2591: Screenshot -->
+
+## Findings {: id="item-list"}
+
+The **Security and risk management findings** page displays a filtered list of findings, sorted by status. Use this page to review and prioritize findings and track the progress of your security efforts.
```

Comment:
TODO confirm sorting options. What will be available at launch?

Reply:
they will be available at a second stage. 
right now, we are going to deliver the first overview tab + findings tab with the table that exists on the items page. 
so i would hold this filtering part for now 🙇🏻‍♀️